Packing.cmake

```diff
@@ -0,0 +1,35 @@
+set(CPACK_PACKAGE_NAME "dragonflydb"
```

Comment:
can you please write here in the comments which command should be used to actually create the package. 
Also, in the PR description as well - for the protocol. i.e. 
"After building dragonfly, run 

```
cpack ....
```
and the deb package will be created in bla...

Reply:
Added